common/djangoapps/student/tests/tests.py

```diff
@@ -1152,5 +1152,5 @@ def _assert_responses(self, response, count):
         self.assertContains(response, 'XSeries Program Course', count)
         self.assertContains(response, 'XSeries Program: Interested in more courses in this subject?', count)
         self.assertContains(response, 'This course is 1 of 3 courses in the', count)
-        self.assertContains(response, self.program_name, count)
+        self.assertContains(response, self.program_name, count * 2 if count else 0)
```

Comment:
Every program name will be shown double now with every course. That's why count would be multiple by 2.
I have added ` <span class="sr">${program_data['display_name']}</span>` within the Xseries button.